src/pages/evm/wallet/SendPage.vue

```diff
@@ -100,6 +107,28 @@ export default defineComponent({
         showContractLink(): boolean {
             return !!this.token?.address;
         },
+        currencyInputSecondaryCurrencyBindings() {
+            const useSecondaryCurrency = !!this.fiatConversionRate;
+
+            if (!useSecondaryCurrency) {
+                return {};
+            }
+
+            return {
+                secondaryCurrencyCode: userStore.fiatCurrency,
+                secondaryCurrencyConversionFactor: this.fiatConversionRate,
+                secondaryCurrencyDecimals: 2,
+            };
+        },
+        fiatLocale() {
+            return userStore.fiatLocale;
+        },
+        tokenSymbol() {
+            return this.token?.symbol ?? 'TLOS';
```

Comment:
I would suggest keeping it as generic as we can for other blockchains.

```suggestion
            return this.token?.symbol ?? chainStore.loggedChain.settings.getSystemToken().symbol;
```